debian/build-hooks/H01paxrat_mono-sgen

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# paxrat /usr/bin/mono-sgen before building environment to prevent build failure
+
+apt-get install -y --force-yes paxrat
+apt-get install -y --force-yes mono-runtime-sgen
+paxrat /usr/bin/mono-sgen
```

Comment:
Paxrat should be called without any arguments or with different arguments.

Reply:
Can you or @dma comment on whether no args or different args are preferable, and if different what these should be? 

I'll copy the fix over to the other PRs I have for the mono-sgen fix as well once I know what the right flags are.

Reply:
Assuming the flag is present (it might not be if the package is pulled from stretch where the config is frozen), you would just `paxrat` it with no arguments. This will apply the flags from `/etc/paxrat/paxrat.conf`. If the flag is not present, it can be added to the conf first.

Reply:
Cool, I'll try to get this done over the weekend because I want to test the change before I commit it :)